app/src/main/java/com/github/swent/echo/compose/components/HomeScreen.kt

```diff
@@ -119,6 +119,8 @@ private fun Content(
     // search mode for displaying events
     val searchMode by homeScreenViewModel.searchMode.collectAsState()
 
```

Comment:
The comment will be added in the next pr where i will comment all my code. But it avoids using discover mode, closing the bottom sheet to look at the event, and reopening the bottom sheet which would display filters instead of discover. Concretely, it remembers the last page you openned